src/components/common/Pagination.tsx

```diff
@@ -0,0 +1,39 @@
+interface PaginationProps {
+  currentPage: number;
+  totalPages: number;
+  onPageChange: (page: number) => void;
+  isVisible: boolean;
+}
+
+const Pagination = ({
+  currentPage,
+  totalPages,
+  onPageChange,
+  isVisible,
+}: PaginationProps) => {
+  return (
+    isVisible && (
+      <div className="flex items-center justify-center gap-4 mt-6">
+        <button
+          className="bg-gray-200 text-gray-700 hover:bg-gray-300"
```

Comment:
can it be moved in const? like `commonClassName` `"bg-gray-200 text-gray-700 hover:bg-gray-300"` just to reuse it across buttons?